patterns/1-initial/principles.md

```diff
@@ -0,0 +1,147 @@
+## Title
+
+Explicit InnerSource Principles
+
+## Patlet
+
+InnerSource as a term is circulating among employees. The usual explanation of
+"applying Open Source best practices inside an organisation" does not work well
+though with people lacking an Open Source background. As a remedy the most
+important principles of InnerSource get published widely.
+
+## Problem
+
+The organisation is trying to role out InnerSource at a larger scale. The
+initiative started among Open Source enthusiasts. The goal is now to get buy in
+from people that are lacking Open Source experience. As a result the typical
+slogan of "applying Open Source best practices" is not longer sufficient to
+transport the message of what InnerSource is, which problems it solves and which
+tools it uses for solving these issues.
+
+## Context
+
+In an organisation early experiments have shown that open source collaboration
+best practices can be beneficial. The goal now is to move the initiative to
+teams and individuals lacking a deep background in open source.
+
+The goal now is to clearly communicate the goals of the InnerSource initiative
+as well as a clear path towards achieving these goals.
+
+## Forces
+
+Teams have trouble communicating exactly what the important aspects of
+InnerSource are.
+
+People lacking open source experience fail to understand what it means to bring
+open source best practices into the organisation.
+
+On a daily basis teams trying to follow InnerSource best practices have a hard
+time deciding if what they are doing is inline with general InnerSource values.
+
+## Solution
+
+Those involved with the InnerSource initiative in the organisation need to
+understand two perspectives:
```

Comment:
Is it really that these are perspectives that they should understand, or are you rather advising them to follow these two steps as part of your solution? I think it is the latter :)

Reply:
I do mean understand two perspectives. However reading this after a break of a few weeks I see how the phrasing was unclear. What I mean is that those who should adopt InnerSource need to understand these two perspectives - not the ones originally involved with rolling out InnerSource.

Reply:
The current **Solution** section mostly sounds more like you are talking to "the ones rolling out InnerSource" actually.
However you might have changed your mind there, midway in writing, as it doesn't seem 100% consistent. 😄 

Let me try to make some inline suggestions, and see if those changes increase clarity.

I think that the Solution section should indeed be written for the "the ones rolling out InnerSource", because this pattern would be used by that group, in order to provide more clarity to the rest of the org (the ones that they want to adopt InnerSource).